apps/squareone/src/pages/_error.jsx

```diff
@@ -0,0 +1,17 @@
+import * as Sentry from '@sentry/nextjs';
+import Error from 'next/error';
+
+const CustomErrorComponent = (props) => {
+  return <Error statusCode={props.statusCode} />;
+};
+
+CustomErrorComponent.getInitialProps = async (contextData) => {
+  // In case this is running in a serverless function, await this in order to give Sentry
+  // time to send the error before the lambda exits
+  await Sentry.captureUnderscoreErrorException(contextData);
+
+  // This will contain the status code of the response
+  return Error.getInitialProps(contextData);
+};
+
+export default CustomErrorComponent;
```

Comment:
We can ditch these example files, but it could be nice to have a page we can use to get a quick check on if the Sentry integration is working:

https://data-dev.lsst.cloud/sentry-example-page